### ISSUE TITLE
Python bug on fedora. Program don't launch

### ISSUE THREAD

**Issue by tr1p0p** [axcore/tartube#437]
On latest Fedora 36, on linux, tartube just don't launch and throw a python error.

```
Traceback (most recent call last):
  File "/usr/bin/tartube", line 40, in <module>
    import mainapp
  File "/usr/lib/python3.10/site-packages/tartube/mainapp.py", line 96, in <module>
    import config
  File "/usr/lib/python3.10/site-packages/tartube/config.py", line 56, in <module>
    from matplotlib.backends.backend_gtk3agg import FigureCanvasGTK3Agg
ModuleNotFoundError: No module named 'matplotlib.backends.backend_gtk3agg'
```

**Comment by axcore** [comment]
Looks like a [matplotlib issue](https://matplotlib.org/3.1.1/api/backend_gtk3agg_api.html).

> TODO We'll add this later, importing the gtk3 backends requires an active X-session, which is not compatible with cron jobs.

**Comment by axcore** [comment]
You can try uninstalling matplotlib from your system, since it's an optional dependency.

**Comment by axcore** [comment]
Your problem should have resolved it self now (using today's release, v2.4.165). If not, feel free to reopen this issue.
